crates/sui-core/src/authority.rs

```diff
@@ -776,6 +783,12 @@ impl AuthorityMetrics {
                 "Number of fastpath transactions sent for execution by consensus transaction handler",
                 registry,
             ).unwrap(),
+            consensus_timestamp_bias: register_histogram_with_registry!(
+                "consensus_timestamp_bias",
+                "Bias of consensus timestamp from system time",
```

Comment:
nit: maybe "gap of system time from consensus timestamp", since this is computed as sys time - consensus time?